src/generator/constants.ts

```diff
@@ -1 +1,2 @@
 export const DEFINITIONS_ROOT = '#/definitions/'
+export const JSON_SCHEMA_ID = 'prisma-json-schema'
```

Comment:
Do you think it might be necessary or it might be a nice feature to configure the id via generator settings? Then, JSON_SCHEMA_ID is the default one but another value can be passed in via generator options.

Reply:
I think it would be great. If no id is provided, the library would have no $id (like it is now). And if someone needs an id, he would just need to specify it. I can make another PR for a feature like that next week.